crypto/x509v3/v3_alt.c

```diff
@@ -141,7 +141,9 @@ STACK_OF(CONF_VALUE) *i2v_GENERAL_NAME(X509V3_EXT_METHOD *method,
 int GENERAL_NAME_print(BIO *out, GENERAL_NAME *gen)
 {
     unsigned char *p;
-    int i;
+    char *data;
+    int i, formatted_length;
+    ASN1_STRING *formatted_output;
     switch (gen->type) {
```

Comment:
Blank line required between vars and code

Reply:
I'll add that in